packages/auth/lib/index.d.ts

```diff
@@ -320,7 +320,7 @@ export namespace FirebaseAuthTypes {
     /**
      * Any additional user information assigned to the user.
      */
-    additionalUserInfo?: AdditionalUserInfo;
+    additionalUserInfo?: null | AdditionalUserInfo;
```

Comment:
```suggestion
    additionalUserInfo: null | AdditionalUserInfo;
```

Wouldn't it no longer be optional if its `null` or something?